brayns/common/scene/Model.h

```diff
@@ -261,9 +267,16 @@ class Model
     */
     BRAYNS_API TrianglesMeshMap& getTrianglesMeshes()
     {
+        _trianglesMeshesDirty = true;
         return _trianglesMeshes;
     }
 
+    template <typename T>
+    void updateTrianglesMeshesBounds(const T& value)
+    {
+        _trianglesMeshesBounds.merge(value);
+    }
```

Comment:
Why is this special? I don't see any apparent reason for the treating triangle meshes differently.

Reply:
Again, not changing anything which is not my concern here.

Reply:
Well, since you're modifying the code I thought you would know the answer.

Reply:
I think it would be safe to remove this function and calculate the bounds in the updateBounds function instead. It needs to be tested more though so maybe it should be done in another PR.